### PR TITLE
Fix/64

### LINKED ISSUE #64: Functional programming: specify wheter letters are capitalised or not.

### PR DESCRIPTION
**Pull request by baffelli**
Fixes #64

### CODE REVIEW

File: functional_programming.ipynb [hunk 1]
```diff
@@ -1275,7 +1275,7 @@
     "    Hints:\n",
     "    <ul>\n",
     "        <li>\n",
-    "            The operator <code>%</code> (modulo) gives the remainder of division. <code>a % k</code> is 0 when a is a multiple of k. \n",
+    "            The operator <code>%</code> (modulo) gives the remainder of division. <code>a % k</code> is 0 when a is a multiple of <code>k</code>. \n",
```
Comment (edoardob90) [inline functional_programming.ipynb:1278]:
```suggestion
    "            The operator <code>%</code> (modulo) gives the remainder of division. <code>a % k</code> is 0 when <code>a</code> is a multiple of <code>k</code>. \n",
```

**Review by edoardob90 — CHANGES_REQUESTED** [review]
Just a minor change, for readability

File: functional_programming.ipynb (hunk 1 above)
Comment (baffelli) [reply functional_programming.ipynb]: Thanks, looks good

**Review by yakutovicha — CHANGES_REQUESTED** [review]
Two minor requests.

File: functional_programming.ipynb [hunk 2]
```diff
@@ -1275,7 +1275,7 @@
     "    Hints:\n",
     "    <ul>\n",
     "        <li>\n",
-    "            The operator <code>%</code> (modulo) gives the remainder of division. <code>a % k</code> is 0 when a is a multiple of k. \n",
+    "            The operator <code>%</code> (modulo) gives the remainder of division. <code>a % k</code> is 0 when <code>a</code> is a multiple of <code>k</code>. \n",
```
[the remark this answers is not in the document]
Comment (baffelli) [reply functional_programming.ipynb]: Fixed with a comment in 12604ab. Unfortunately I cannot add internal link using markdown here, I tried with HTML links but it did not work.

**Review by yakutovicha — APPROVED** [review]
LGTM!

**Review by edoardob90 — APPROVED** [review]

**Pull request merged**